### ISSUE TITLE
v6 toolbar background color issue

### ISSUE THREAD

**Issue by ksuuk** [hazuki0x0/YuzuBrowser#187]
Since v6, the bottom toolbar -s dark background is white and the buttons are invisible, however when I open settings menu, then there the toolbar -s background is dark again, so theme is correct:

Screenshots added.
![Screenshot 2021-02-15 at 09 59 31](https://user-images.githubusercontent.com/26193002/107920013-e68fb800-6f74-11eb-9c34-12b9bf20d5d7.png)

![Screenshot 2021-02-15 at 09 59 17](https://user-images.githubusercontent.com/26193002/107920024-ea233f00-6f74-11eb-97ff-b417f02a792c.png)

**Comment by hazuki0x0** [comment]
What is your device and android version?

**Comment by ksuuk** [comment]
> What is your device and android version?

Xperia X, stock Oreo 8.0.

BTW, is this normal, that after v5 > v6 upgrade, Yuzu didn't use find custom theme and used default theme and user must import and select custom theme again?

**Comment by hazuki0x0** [comment]
> BTW, is this normal, that after v5 > v6 upgrade, Yuzu didn't use find custom theme and used default theme and user must import and select custom theme again?

Yes, you need to import custom theme due to scoped storage limitations (android 10 or higher). Of course it doesn't affect older versions of android, but I changed it for consistency.

**Comment by ksuuk** [comment]
Tested more, made new, dark based custom theme, copied all settings/colors from old light to the new dark theme. When I compared old and new theme settings (to find out what causes the toolbar background color issue) turned out, that settings are now identical:

{
  "lightTheme": "true",
  "showTabDivider": "true",
  "pullToRefreshDark": "true",
  "toolbarBackgroundColor": "0xffffffff",
  "toolbarTextColor": "0xff080808",
  "toolbarImageColor": "0xff3069c4",
  "qcItemBackgroundColorNormal": "0xf4c57373",
  "qcItemBackgroundColorSelect": "0xffef8881",
  "statusBarDarkIcon": "true"
}

But Yuzu is showing them differently and also Yuzu Browser Theme Creator -s theme preview doesn't match the Yuzu layout. 

![Screenshot 2021-02-15 at 08 57 42](https://user-images.githubusercontent.com/26193002/107915749-bbee3100-6f6d-11eb-9668-9e57570cdd51.png)
![Screenshot 2021-02-15 at 09 10 36](https://user-images.githubusercontent.com/26193002/107915771-c6102f80-6f6d-11eb-9eae-8f54d0fddf09.png)

So seems that browser and editor are not compatible and Yuzu ignores some custom theme settings.

When I select built in light theme, the toolbar is grey:

![Screenshot 2021-02-15 at 09 07 22](https://user-images.githubusercontent.com/26193002/107915823-e2ac6780-6f6d-11eb-8a00-ca94bfd0c9c8.png)

And after using built in light or dark theme and then selecting custom theme, Yuzu crashes and looses all browser settings and uses default layout:

![Screenshot 2021-02-15 at 09 06 06](https://user-images.githubusercontent.com/26193002/107915932-12f40600-6f6e-11eb-8a62-3ea782431922.png)
